app/models/service_orchestration.rb

```diff
@@ -77,6 +78,11 @@ def all_vms
     orchestration_stack.vms
   end
 
+  def add_resource(rsc, _options = {})
+    raise "Service Orchestration subclass does not support add_resource for #{rsc.class.name}" unless rsc.kind_of?(OrchestrationStack)
```

Comment:
@d-m-u This would break the Orchestration Service.
```
Error: Service Orchestration subclass does not support add_resource for ManageIQ::Providers::Azure::CloudManager::OrchestrationTemplate
```

Reply:
@lfu Thanks we were limiting what can be connected to an Orchestration Service object.  We can expand this list, but it should be minimal since created resources would get connected through the Stack object.

Would have have a link to where the OrchestrationTemplate gets attached?